test/e2e/quadlet_test.go

```diff
@@ -859,6 +861,10 @@ BOGUS=foo
 		Entry("image-no-image.volume", "image-no-image.volume", 1, "converting \"image-no-image.volume\": the key Image is mandatory when using the image driver"),
 		Entry("Volume - global args", "globalargs.volume", 0, ""),
 		Entry("Volume - Containers Conf Modules", "containersconfmodule.volume", 0, ""),
+		Entry("Volume - Quadlet image (.build)", "build.quadlet.volume", 0, ""),
```

Comment:
This test (and the similar one for the `.image` file) is currently failing, even though it works when I am manually running `quadlet` on the test file. The error message during test is:

```
> Enter [It] Volume - Quadlet image (.build) - /var/home/jm/go/src/github.com/containers/podman/test/e2e/quadlet_test.go:864 @ 05/16/24 15:26:59.847
  Running: /var/home/jm/go/src/github.com/containers/podman/bin/quadlet --user --no-kmsg-log /tmp/podman-e2e-1875717062/subtest-2494887151/generated with QUADLET_UNIT_DIRS=/tmp/podman-e2e-1875717062/subtest-2494887151/quadlet
  quadlet-generator[122417]: converting "build.quadlet.volume": requested Quadlet image basic.build was not found
  [FAILED] Failure recorded during attempt 1:
  Expected
      <int>: 1
  to match exit code:
      <int>: 0
```

Yet, `basic.build` and `basic.image` are existing.

Reply:
Is it possible to keep the temp directories the test suite creates?

Reply:
I have fixed this issue by adding a new testcase type: "Running quadlet test case with dependencies"